src/style.css

```diff
@@ -0,0 +1,43 @@
+* {
```

Comment:
As in the header example try to use something like [Meyer Reset](https://meyerweb.com/eric/tools/css/reset/), you don't have control over what tags you reset by using *. I suggest creating a separate css file for the sake of reseting and importing it to this file